sdk/tables/azure-data-tables/tests/test_table_cosmos_async.py

```diff
@@ -99,7 +99,7 @@ async def test_create_table_fail_on_exist(self, tables_cosmos_account_name, tabl
     async def test_query_tables_per_page(self, tables_cosmos_account_name, tables_primary_cosmos_account_key):
         # Arrange
         # account_url = self.account_url(tables_cosmos_account_name, "table")
-        # ts = self.create_client_from_credential(TableServiceClient, tables_primary_cosmos_account_key, account_url=account_url)
+        # ts = self.create_client_from_credential(TableServiceClient, tables_primary_cosmos_account_key, endpoint=account_url)
```

Comment:
I know you probably did a find and replace, but I think we can get rid of these comments now.